### PR TITLE
clean up eslint issues

### PR DESCRIPTION
**Pull request by moloko**
There were so many (355!) eslint issues that just viewing this file was hurting my eyes, with so much error highlighting! All should now be fixed.

When reviewing: please note, the aim of this was to do just enough to clear the eslint errors. I'm sure you will spot plenty of ways this could be further improved but really I would prefer to do that as a separate job!

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/adapt-contrib-media.js [hunk 1]
```diff
@@ -38,31 +37,31 @@ define([
    * is a duration integer the other is the player object. The default functions error on slider key press and so break
    * accessibility. Below is a correction.
    */
-  _.extend(mejs.MepDefaults, {
+  _.extend(window.mejs.MepDefaults, {
```
Comment (tomgreenfield) [inline js/adapt-contrib-media.js:40]: Toying instead with the idea of having a nested `.eslintrc.json` at plugin level i.e.:
```json
{
  "globals": {
    "mejs": true
  }
}
```
Would then nicely match what we do with known libraries at [framework level](https://github.com/adaptlearning/adapt_framework/blob/fb48d28115a357d71cbf3259be9c270cdcff77fa/.eslintrc.json#L11-L22).

Comment (moloko) [reply js/adapt-contrib-media.js:40]: yeah could be a good idea. I still like to include `window.` anyway though.

**Review by tomgreenfield — APPROVED** [review]
No refusal from me 👁️

**Comment by moloko** [comment]
thanks @tomgreenfield !

**Pull request merged**